workers/unity/Packages/com.improbable.gdk.core/Utility/PlayerLoopUtils.cs

```diff
@@ -0,0 +1,238 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Unity.Entities;
+using UnityEngine;
+using UnityEngine.Experimental.LowLevel;
+using UnityEngine.Experimental.PlayerLoop;
+
+namespace Improbable.Gdk.Core
+{
+    public static class PlayerLoopUtils
+    {
+        [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+        public class UpdateInSubSystemAttribute : Attribute
+        {
+            public UpdateInSubSystemAttribute(Type subSystemType)
+            {
+                SubSystemType = subSystemType;
+            }
+
+            public Type SubSystemType { get; }
+        }
+
+        // HACK: We need to call this specific private method to ensure the EntityDebugger shows our groups in the PlayerLoop
+        // No need to cache these methods, as each type will almost always be called once.
```

Comment:
When won't it be called once?

Reply:
If worlds share a system, then its called twice ;)

Reply:
Share as in a reference to the same instance or each world has a copy of it? I assume the latter?